draft-ietf-acme-acme.md

```diff
@@ -1428,6 +1436,35 @@ over the same type, e.g., ["http-03", "http-05"].  Challenges within a type are
 testing the same capability of the domain owner, and it may not be possible to
 satisfy both at once. ]]
 
+## Validation of Wildcard Authorizations
+
+When validating a challenge other than a proof of possesion challenge which is
+part of an authorization for a wildcard name (a domain name which contains one
+or more `*` labels), the challenge is validated as follows:
+
+- A CA chooses some positive integer n by policy.
+
+- The CA generates n random strings for each of the m `*` labels. Let `s(i,j)`
+  denote string i for `*`-label j. Each string MUST consist of lowercase
+  letters, decimal digits and hyphens and MUST NOT exceed 63 characters. Each
```

Comment:
This should say that it MUST be a valid DNS label and SHOULD not start with `xn--`.  Pure random generation could end up with starting or ending with `-` which is not valid and could end up looking like a punycode label.